### ISSUE TITLE
Slow response and timeouts from https://letterboxd-list-radarr.onrender.com

### ISSUE THREAD

**Issue by gingerbeardman** [screeny05/letterboxd-list-radarr#36]
For the past couple days I'm getting timeout errors from the list URL.

Requesting it manually I see it's very slow.

**Comment by screeny05** [comment]
For me the service is pretty snappy. Of course, the server runs with only 25mb of redis-cache, so maybe the lists you request are really large and/or not cached.

Unfortunately render.com doesn't give me good metrics for figuring out what's going on.

I just did a manual restart of the service. Maybe that'll solve it.

**Comment by screeny05** [comment]
Yeah, running it locally is the easiest way to achieve good performance. If you encounter any issues with the docker-image, just let me know.
